static/javascript/launch.js

```diff
@@ -621,12 +635,28 @@ function populateDropDownWithValue(selector, value) {
   }
 }
 
+function setTabIndex(metadataDetail, value) {
+  document.getElementById(metadataDetail).tabIndex = value;
+}
+
+function initialiseTabIndex() {
+  const details = [
+    "cir_metadata_detail",
+    "survey_type_metadata_detail",
+    "sds_metadata_detail",
+  ];
+  for (i = 0; i < 3; i++) {
```

Comment:
Minor - If the `details` array ever changes, the hardcoded `3` would also need to be changed. Could use something like `details.length` instead? Probably a super cool JS way of doing it but I'm not sure 😆

Reply:
Updated 👍